saleor/webhook/payloads.py

```diff
@@ -1069,3 +1069,55 @@ def generate_excluded_shipping_methods_for_checkout_payload(
         ],
     }
     return json.dumps(payload, cls=CustomJsonEncoder)
+
+
+@traced_payload_generator
+def generate_payment_action_request_payload(
+    payment_data: "PaymentActionData",
+    requestor: Optional["RequestorOrLazyObject"] = None,
+) -> str:
+    payment = payment_data.payment
+
+    action_value = (
+        quantize_price(payment_data.action_value, payment.currency)
+        if payment_data.action_value
+        else None
+    )
+
+    order_id = payment.order_id
+    graphql_order_id = (
+        graphene.Node.to_global_id("Order", order_id) if order_id else None
+    )
```

Comment:
On `main` we recently added `to_global_id_or_none`, you may consider using it here (or we can add later before merging feature branch to main).

Reply:
@maarcingebala, 
Hhm, `to_global_id_or_none` takes an instance. In that case, I operate only on IDs. `payment.order_id`. If I will switch to this approach, I will have to call `payment.order` which means, that this could add an additional query to fetch an order/checkout instance. Are you ok with this?

Reply:
OK so ignore my comment then, we can keep your solution in this case and save the database query.